src/Storages/ProjectionsDescription.h

```diff
@@ -28,6 +28,8 @@ struct ProjectionDescription
         Aggregate,
     };
 
+    static constexpr const char * VIRTUAL_PROJECTION_NAME = "_minmax_projection";
```

Comment:
Let's assume in future we could have more virtual projections.
Maybe we'd better add a flag `is_virtual` or something, and check it instead of comparing string all the time ...
Or specific flag `is_minmax_projection`.